### ISSUE TITLE
curl (25) error 

### ISSUE THREAD

**Issue by bwgartner** [linuxhw/hw-probe#32]
On openSUSE Tumbleweed ( hw-probe package installed from http://download.opensuse.org/repositories/hardware/openSUSE_Tumbleweed/ ), the following error occurs:

```
hw-probe -probe -upload
Probe for hardware ... Ok
curl: (25) Chunky upload is not supported by HTTP 1.0
ERROR: failed to upload data, curl error code "25"
```

Installing the openSUSE Leap15 version ( from respective repo ) worked fine (on the same hardware moments before).

**Comment by lvc** [comment]
Hi,

Thanks for reporting this.

But I can't reproduce it on the latest Tumbleweed ISO (openSUSE-Tumbleweed-GNOME-Live-x86_64-Snapshot20190115-Media.iso).

What is your ISO build number, curl version and arch?

Thank you.

**Comment by bwgartner** [comment]
FYI, this was a fresh install of openSUSE-20190115, but only a minimal "base" install.

Answers to your queries, respectively ...
- openSUSE-Tumbleweed-NET-x86_64-Snapshot20190115-Media.is
- # curl --version
curl 7.63.0 (x86_64-suse-linux-gnu) libcurl/7.63.0 OpenSSL/1.1.0h zlib/1.2.11 libidn2/2.1.0 libpsl/0.20.2 (+libidn2/2.1.0) libssh/0.8.6/openssl/zlib nghttp2/1.34.0
Release-Date: 2018-12-12
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtsp scp sftp smb smbs smtp smtps telnet tftp 
Features: AsynchDNS IDN IPv6 Largefile GSS-API Kerberos SPNEGO NTLM NTLM_WB SSL libz TLS-SRP HTTP2 UnixSockets HTTPS-proxy PSL Metalink
- x86_64

FYI, after I just rebooted the system to gather the above info, I can no longer reproduce the error either. See https://linux-hardware.org/?probe=c87500a262 ... so feel free to close this issue for now.

**Comment by lvc** [comment]
Let's drop curl --http1.0 option.

Fixed by d6b5278fa9bb347d124159dd966c2ff58324eadd.

Thank you.
